x-pack/plugins/ml/readme.md

```diff
@@ -0,0 +1,137 @@
+# Documentation for ML UI developers
+
+## Setup local environment
+
+### Kibana
+
+1. Fork and clone the [Kibana repo](https://github.com/elastic/kibana).
+
+1. Install `nvm`, `node`, `yarn` (for example, by using Homebrew). See 
+   [Install dependencies](https://www.elastic.co/guide/en/kibana/master/development-getting-started.html#_install_dependencies).
+
+1. Make sure that Elasticsearch is deployed and running on localhost:9200.
+
+1. Navigate to the directory of the `kibana` repository on your machine.
+
+1. Fetch the latest changes from the repository.
+
+1. Checkout the branch of the version you want to use. For example, if you want 
+   to use a 7.9 version, run `git checkout 7.9`.
+
+1. Run `nvm use`. The response shows the Node version that the environment uses. 
+   If you need to update your Node version, the response message contains the 
+   command you need to run to do it.
+
+1. Run `yarn kbn bootstrap`. It takes all the dependencies in the code and 
+   installs/checks them. It is recommended to use it every time when you switch 
+   between branches.
+
```

Comment:
Can we just use [this link](https://www.elastic.co/guide/en/kibana/master/development-getting-started.html) instead? It's general setup steps for Kibana and in case something changes we don't need to update it

Reply:
@darnautov Sure, we can! Thanks for the link.

Reply:
I think it's useful to have the quick setup instructions here too, but definitely worth adding the link in too.